api/src/main/resources/client_api.yaml

```diff
@@ -126,6 +126,31 @@ paths:
           schema:
             $ref: "#/definitions/ConceptListResponse"
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/domainCounts:
+    post:
+      tags:
+        - concepts
+      consumes:
+        - application/json
+      description: >
+        Searches for concepts in concept table by name, and optionally filter on
+        domain, vocabulary IDs, or standard concept status. Uses the CDR version affiliated
+        with the workspace specified in the path.
+      operationId: "domainCounts"
+      parameters:
+        - $ref: '#/parameters/workspaceNamespace'
+        - $ref: '#/parameters/workspaceId'
+        - in: body
+          name: request
+          description: domain counts search request
+          schema:
+            $ref: "#/definitions/DomainCountsRequest"
```

Comment:
@dolbeew Should this api call be lumped in the the domain data api call, since we will only we showing 1 domain per bucket click?

Reply:
SGTM. And when searching from the homepage (all tabs), we would call that combined endpoint once for each tab?

Reply:
Maybe, we can discuss this more after public launch.